React/tic-tac/.babelrc

```diff
@@ -13,7 +13,7 @@
           "@utils": "./src/utils",
           "@config": "./src/config",
           "@services": "./src/services",
-          "@login": "./src/screens/Login"
+          "@header": "./src/Header"
```

Comment:
I think this alias doesn't work. Isn't `Header` inside the `app/components` folder?